modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/extra/remote-app/src/common/services/request.ts

```diff
@@ -10,12 +9,180 @@
  * distribution rights of the Software.
  */
 
+import dayjs from 'dayjs';
+
+import {FIELDSREPORT, RequestFilterType} from '../../types';
 import fetcher from './fetcher';
+import {
+	filteredOrganizationsERC,
+	getIdGreaterOrEqualOrganization,
+	getIdLessOrEqualOrganization,
+	getOrganizationBetweenIds,
+	getOrganizationName,
+} from './organization';
 
 const resource = 'o/c/evprequests';
 
-export async function getRequestsByFilter() {
-	const response = await fetcher(`${resource}`);
+const nestedFields = '&nestedFields=r_organization_c_evpOrganization';
+
+const formatArrayUrl = (key: string, value: String[]) => {
+	let operator = `${key} in ({values})`;
+
+	operator = operator
+		.replace('{values}', value.map((value) => `'${value}'`).join(','))
```

Comment:
Could you put the type to value parameter?
```suggestion
		.replace('{values}', value.map((value: ???) => `'${value}'`).join(','))
```